asf_heat_pump_suitability/pipeline/run/heat_network_city_centres.py

```diff
@@ -78,7 +80,45 @@ def parse_arguments() -> argparse.Namespace:
             ["UPRN", "LAD23NM", "X_COORDINATE", "Y_COORDINATE", "in_hn_zone", "ZoneID"]
         ).rename({"ZoneID": "HNZoneID"})
 
-        # TODO filter and add label for UPRNs in city centres
+        """ City centre areas """
+
+        # Load spatial signature polygons
+        print("Loading spatial signatures dataset...")
+        spatial_signatures_gb_simplified_gdf = (
+            load_geodata.load_gdf_spatial_signatures_gb(detail_level="simplified")
+        )
+
+        # Label UPRNs in city centres
+        city_centre_types = [  # TODO confirm types
```

Comment:
```suggestion
        city_centre_types = [  # TODO confirm types with scaling
```
I think this is especially important!